### DISCUSSION TITLE
Could not load the "sharp" module

### DISCUSSION THREAD

**Discussion by siamakabi** [nuxt/nuxt#25589] [Questions] [answered]
Hello
This error is shown in the photo link:

`{"error":{"message":"[500] [IPX_ERROR] Could not load the "sharp" module using the linux-x64 runtime\nPossible solutions:\n- Ensure optional dependencies can be installed:\n npm install --include=optional sharp\n yarn add sharp --ignore-engines\n- Ensure your package manager supports multi-platform installation:\n See https://sharp.pixelplumbing.com/install#cross-platform\n- Add platform-specific dependencies:\n npm install --os=linux --cpu=x64 sharp\n- Consult the installation documentation:\n See [https://sharp.pixelplumbing.com/install"}}](https://sharp.pixelplumbing.com/install%22%7D%7D)`

**Comment by TheAlexLichter** [comment, marked as answer]
https://github.com/nuxt/image/issues/1210#issuecomment-1917921546
